kuma/javascript/src/user-provider.test.js

```diff
@@ -1,11 +1,14 @@
 //@flow
 /* eslint-disable camelcase */
 import React from 'react';
-import { act, create } from 'react-test-renderer';
+import { render, waitFor } from '@testing-library/react';
 
 import GAProvider from './ga-provider.jsx';
 import UserProvider from './user-provider.jsx';
 
+// This can be removed once https://github.com/mdn/kuma/pull/6940 lands
+require('regenerator-runtime/runtime');
```

Comment:
Let's wait till #6940 lands so we don't have to remember to make another PR just to remove this.